ds/eurito_indicators/indicators/make_articles_final.py

```diff
@@ -238,61 +264,55 @@ def make_schema(indicator_type, category, suffix="count"):
         schema["schema"]["value"]["id"] = f"{category}_{suffix}"
     elif indicator_type == "clusters":
         schema = fetch_template_schema("articles_clusters_base")
-        #schema["date"] = str(datetime.date.today())
-        schema['data_date'] = "2021/07/01"
+        # schema["date"] = str(datetime.date.today())
+        schema["data_date"] = "2021/07/01"
         schema["is_experimental"] = True
-        schema[
-            "title"
-        ] = f"{CLEAN_CLUSTERS[category]} preprints (count)"
+        schema["title"] = f"{CLEAN_CLUSTERS[category]} preprints (count)"
         schema[
             "subtitle"
-        ] = f"Number participations in {CLEAN_CLUSTERS[category]} preprints by institutions in the location (research clusters identified through a topic model of article abstracts)"
+        ] = f"Number participations in {CLEAN_CLUSTERS[category]} preprints by institutions in the location"
```

Comment:
I think `Number participations` should be `Number of participations` everywhere in this script